frontend/admin/src/js/components/searchRequest/SearchRequestTable.tsx

```diff
@@ -1,44 +1,55 @@
-import React, { useMemo } from "react";
+import React, { useCallback, useMemo } from "react";
 import { IntlShape, useIntl } from "react-intl";
 import { notEmpty } from "@common/helpers/util";
-import { FromArray } from "@common/types/utilityTypes";
-import { getLocale } from "@common/helpers/localize";
+import { getLocalizedName } from "@common/helpers/localize";
 import { getPoolCandidateSearchStatus } from "@common/constants/localizedConstants";
 import { PoolCandidateSearchStatus } from "@common/api/generated";
 import Pending from "@common/components/Pending";
 import { getFullPoolAdvertisementTitle } from "@common/helpers/poolUtils";
 import { formatDate, parseDateTimeUtc } from "@common/helpers/dateUtils";
 import {
-  GetPoolCandidateSearchRequestsQuery,
+  ApplicantFilter,
+  Maybe,
+  PoolCandidateFilter,
+  PoolCandidateSearchRequest,
   useGetPoolCandidateSearchRequestsQuery,
 } from "../../api/generated";
 import Table, { ColumnsOf, tableViewItemButtonAccessor } from "../Table";
 import { useAdminRoutes } from "../../adminRoutes";
 
-type Data = NonNullable<
-  FromArray<GetPoolCandidateSearchRequestsQuery["poolCandidateSearchRequests"]>
->;
+interface IRow {
+  original: {
+    poolCandidateFilter?: PoolCandidateFilter;
+    applicantFilter?: ApplicantFilter;
+  };
+}
 
 // callbacks extracted to separate function to stabilize memoized component
 const statusAccessor = (
   status: PoolCandidateSearchStatus | null | undefined,
   intl: IntlShape,
-) => (
-  <span>
-    {status
-      ? intl.formatMessage(getPoolCandidateSearchStatus(status as string))
-      : ""}
-  </span>
-);
+) =>
+  status
+    ? intl.formatMessage(getPoolCandidateSearchStatus(status as string))
+    : "";
```

Comment:
statuses always appear blank

it appears to be due to this highlighted section that still uses `PoolCandidateSearchStatus`
but the `status` field was dropped in favour of `done_at` recently in this PR #4817 

![image](https://user-images.githubusercontent.com/40485260/203608950-fe1be534-b0b1-423e-905a-590eab4b2c16.png)

Reply:
oh, I see the issue, you really need to update from main
please do so

![image](https://user-images.githubusercontent.com/40485260/203610355-8eae7910-5a77-4f73-938b-427f9c64dd5b.png)